README.md

```diff
@@ -7,6 +7,7 @@ An Ansible role for installing Node.js.
 - `nodejs_version` - Node.js version (default: `0.10.32`)
 - `nodejs_os` - Node.js operating system build (default: `linux`)
 - `nodejs_arch`: Node.js architecture build (default: `x64`)
+- `npm_version`: npm version (default: `1.4.28`)
```

Comment:
I think that this should be prefixed with `nodejs_` despite being an NPM specific setting.